src/char_data/mod.rs

```diff
@@ -112,9 +112,9 @@ mod tests {
         assert_eq!(bidi_class('\u{FB1D}'), R);
         assert_eq!(bidi_class('\u{FB4F}'), R);
         assert_eq!(bidi_class('\u{FB50}'), AL);
-        assert_eq!(bidi_class('\u{FDCF}'), AL);
+        assert_eq!(bidi_class('\u{FDCF}'), ON);
         assert_eq!(bidi_class('\u{FDF0}'), AL);
-        assert_eq!(bidi_class('\u{FDFF}'), AL);
+        assert_eq!(bidi_class('\u{FDFF}'), ON);
         assert_eq!(bidi_class('\u{FE70}'), AL);
         assert_eq!(bidi_class('\u{FEFE}'), AL);
```

Comment:
why did you changed this ?

Reply:
I verified it against the current data, the test was failing